spec/dns/named_spec.rb

```diff
@@ -74,7 +75,8 @@
     before :each do
       allow(named).to receive(:setup?).and_return(true)
       # Prevents restart_named from accidentally running, DANGEROUS
```

Comment:
Should this comment be changed refer to `start_named` now? Also, can we change it to mention what is 'DANGEROUS', similar to https://github.com/alces-software/metalware/pull/174/commits/f934d314a5810d669cd7a71c0100d38e2f20af9d#diff-a753d91681a41b66f585bed90c0db93fR53?